pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -23,6 +23,7 @@ import cockpit from 'cockpit';
 import VmOverviewTab, { commonTitles } from './vmOverviewTab.jsx';
 import VmLastMessage from './vmLastMessage.jsx';
 import { VCPUModal } from './vcpuModal.jsx';
+import MemModal from './vm/memModal.jsx';
```

Comment:
Do you mind renaming the component to MemoryModal and the file to memoryModal.jsx? Thanks!